cmake/FindSanitizers.cmake

```diff
@@ -20,54 +20,63 @@
 # Copyright (C) 2018 Scylladb, Ltd.
 #
 
-include (CheckCXXSourceCompiles)
+if(NOT Sanitizers_FIND_COMPONENTS)
+  set(Sanitizers_FIND_COMPONENTS
+    address
+    undefined_behavior)
+endif()
```

Comment:
this preserves the backward compatibility if some parent projects use `find_package(Sanitizers)` without specifying the `COMPONENTS` parameter.